### ISSUE TITLE
kafka

### ISSUE THREAD

**Issue by aloon** [scm-spain/howto-spring-cloud#13]
I miss an example integrating kafka

**Comment by fiunchinho** [comment]
Yep. Kafka is still missing. If you need it right now, kafka is being used inside [the zipkin consumer example](https://github.com/scm-spain/howto-spring-cloud/blob/master/zipkin-consumer/src/main/java/com/scmspain/howtospring/zipkin/Consumer.java)
